linked_list/linked_list.py

```diff
@@ -23,13 +23,39 @@ def __init__(self):
     self.tail = None
 
   def add_to_tail(self, value):
-    pass
+    new_node = Node(value)
+    if self.head is None:
+      self.head = new_node
+      self.tail = new_node
+    else:
+      self.tail.set_next(new_node)
+      # self.tail.set_next(None)
+    self.tail = new_node
 
   def remove_head(self):
-    pass
+    curr_head = self.head
+    if self.head:
+      self.head = self.head.get_next()
+      self.tail = None
+      return curr_head.value
```

Comment:
Well done with this implementation; this tends to be one of the trickier parts of this week's repo.